docs/transition.html

```diff
@@ -61,6 +61,8 @@
       gogo-serial extension.  Change your model to use gogo-serial as opposed to
       gogo.
     <p>
+      More details [here](https://github.com/NetLogo/NetLogo/wiki/GoGo-Upgrade).
```

Comment:
It doesn't need a separate `<p>` tag. It can go just after the prior sentence ends.